charts/liquibase-migration/templates/_job.yaml

```diff
@@ -0,0 +1,102 @@
+{{- define "liquibase-migration.job" -}}
+{{- if .Values.migration.enabled }}
+{{- $imageName := .Values.migration.imageName | required "A valid migration.imageName is required (without trailing tag)" -}}
+{{- $jarLocation := .Values.migration.jarLocation | required "A valid migration.jarLocation is required (read by migration.appContainerShell)" -}}
+{{- $secretPrefix := .Values.migration.secretPrefix | required "A valid migration.secretPrefix is required (usually some deploymentDefaults.name)" -}}
+{{- $appName := .Values.migration.appName | default .Release.Name -}}
+{{- $labelRef := .Values.migration.labelRef | default (printf "%s.labels" $appName) -}}
+{{- $syncWave := .Values.migration.syncWave | default "-1" -}}
+{{- $imageTag := .Values.migration.imageTag | default .Values.global.applicationVersion -}}
+{{- $serviceAccount := .Values.migration.serviceAccount | default (printf "%s-sa" $appName) -}}
+{{- $appContainerShell := .Values.migration.appContainerShell | default "bash" -}}
+{{- $liquibaseCommand := .Values.migration.liquibaseCommand | default "update" -}}
+{{- $liquibaseExitCode := .Values.migration.ignoreLiquibaseFailures | default false | ternary "0" "$?" -}}
```

Comment:
Setting defaults here might be redundant if you are already setting them in `values.yaml`.

Reply:
The values.yaml file is a no-op, unfortunately. I put a comment at the top of the file trying to give that heads up, but happy for any other ideas on how to communicate that. It isn't read by anything since this is a library chart. Other library charts I've seen still include a values.yaml for doc purposes, but all the defaults have to be encoded in the chart like this.